pkg/admission/util_test.go

```diff
@@ -263,18 +263,16 @@ func TestGenerateAppID(t *testing.T) {
 		conf.AMFilteringGenerateUniqueAppIds: fmt.Sprintf("%t", true),
 	})
 
-	podUid := "abcd1234-5678-efgh-90ij-klmnopqrstuv"
-
 	// short namespace name
 	ns := "short"
-	appID = generateAppID(ns, createTestingPodWithCustomFields(ns), uniqueEnabled.GetGenerateUniqueAppIds())
```

Comment:
This leaves the function `createTestingPodWithCustomFields()` unused and we should remove it.

Reply:
The lint check will probably fail due to this unused function